lib/public/views/Environments/ColoredEnvironmentStatusComponent.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+
+/**
+ * Change the color of the status of the environment based off which status
+ *
+ * @param {String} status the status to check, format and return
```

Comment:
`String` => `string`, the docblock seems to be outdated? Maybe `the environment status to display`